docs/source/en/main_classes/executorch.md

```diff
@@ -0,0 +1,32 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
```

Comment:
@guangy10 

We should add the following here 

"Copyright (c) Meta Platforms, Inc. and affiliates."

cc @amyeroberts @ArthurZucker

Reply:
Also should "HuggingFace Team" be here since you technically haven't contributed to this file yet.